### ISSUE TITLE
COPYRIGHT

### ISSUE THREAD

**Issue by terrayi** [libhangul/libhangul#62]
libhangul 과 libhangul 을 이용해 개발한 BeOS/Haiku 용 한글 입력기를 Haiku 용으로 패키징 작업 중입니다.
COPYRIGHT 문구가 필요한데 깃헙에는 아무리 찾아보아도 보이지 않네요.
뭐라고 적으면 좋을지 알려주시거나 COPYRIGHT 파일을 추가해주시면 감사하겠습니다.

**Comment by choehwanjin** [comment]
혹시 https://github.com/libhangul/libhangul/blob/main/COPYING 파일을 찾으시는 건가요?

**Comment by terrayi** [comment]
특별히 프로젝트 자체 카피라이트는 없나요?

**Comment by choehwanjin** [comment]
> 특별히 프로젝트 자체 카피라이트는 없나요?

별다른 것은 없습니다.
[hanja.txt](https://github.com/libhangul/libhangul/blob/main/data/hanja/hanja.txt), [mssymbol.txt](https://github.com/libhangul/libhangul/blob/main/data/hanja/mssymbol.txt) 파일은 [BSD-3-Clause 라이선스](https://opensource.org/licenses/BSD-3-Clause) 입니다.
나머지는 LGPL 입니다.

**Comment by terrayi** [comment]
네 감사합니다. HaikuPorts 에 머지되면 알려드리겠습니다.

**Comment by Begasus** [comment]
Looking through the copyright notices in the files: https://github.com/libhangul/libhangul/search?p=1&q=copyright&type=code
I think this should be `2004-2021 Choe Hwanjin`?

**Comment by choehwanjin** [comment]
> Looking through the copyright notices in the files: https://github.com/libhangul/libhangul/search?p=1&q=copyright&type=code I think this should be `2004-2021 Choe Hwanjin`?

Yes, you are right about some files. But I am so lazy to update them.
And I think that there is no need to do it diligently.

**Comment by Begasus** [comment]
> > Looking through the copyright notices in the files: https://github.com/libhangul/libhangul/search?p=1&q=copyright&type=code I think this should be `2004-2021 Choe Hwanjin`?
> 
> Yes, you are right about some files. But I am so lazy to update them. And I think that there is no need to do it diligently.

Not asking you to, fair enough, I think using the time-frame `2004-2021` should cover you work here and in our recipe at haikuports. :)

**Comment by terrayi** [comment]
HaikuPorts 에 대한 미흡한 이해로 처음 작업을 하다보니 예상 외로 시간이 걸렸지만 드디어 머지가 되었습니다.

**Comment by Aquino55** [comment]
😊
